ecosystem/lava-sdk/src/lavasession/consumerTypes.ts

```diff
@@ -217,7 +217,7 @@ export class SingleConsumerSession {
       this.qoSInfo.lastQoSReport.setSync(sync.toFixed());
 
       if (BigNumber(1).gt(sync)) {
-        Logger.debug(
+        Logger.info(
```

Comment:
let's keep these as debug, since they are quite spammy and show only internal usage information